coursier.bzl

```diff
@@ -200,6 +200,16 @@ def get_netrc_lines_from_entries(netrc_entries):
                 netrc_lines.append("password {}".format(password))
     return netrc_lines
 
+def get_home_netrc_contents(repository_ctx):
+    # Copied with a ctx -> repository_ctx rename from tools/build_defs/repo/http.bzl's _get_auth.
+    # Need to keep updated with improvements in source since we cannot load private methods.
```

Comment:
please apply buildifier on this file. a couple of indents look off.

Reply:
Done. FYI: running buildifier on the whole repo creates many changes. Would it be OK to do a buildifier run on the whole repo and commit that? I know that the rules suggest not doing that.

Reply:
Thanks. Yeah, I'm OK with a formatting PR.